src/reimer_arenkiel_lab_to_nwb/dj_utils.py

```diff
@@ -344,11 +302,18 @@ def add_fluorescence(nwbfile, plane_segmentation, key: dict = None, verbose: boo
         fluoresence = nwbfile.processing["ophys"].data_interfaces[f"Fluorescence{segmentation_method}"]
     fluoresence.add_roi_response_series(roi_response_series)
 
+def get_imaging_start_time(key: dict = None):
+    restriction = (
+            odor.MesoMatch & key
+    )  # Since tables can be restrictions, saving this shorthand simplifies following code
+    odor_scan_times = (odor.OdorSync & restriction).fetch1("frame_times")
+    return odor_scan_times[0]
 
-def init_nwbfile(key: dict) -> NWBFile:
+def init_nwbfile(key: dict, metadata: dict = None) -> NWBFile:
     data = (all_sessions.Session & key).fetch1()
+    metadata = metadata or default_ophys_metadata
 
-    nwbfile_kwargs = deepcopy(default_ophys_metadata["NWBFile"])
+    nwbfile_kwargs = deepcopy(metadata["NWBFile"])
 
     nwbfile_kwargs.update(dict(
         session_description=f"animal {data['animal_id']} session {data['session']}",
```

Comment:
Can you add something more descriptive here? E.g. this session contains two photon imaging ...

Reply:
Updated in the yaml file